### PR TITLE
Arrays tasks

### PR DESCRIPTION
**Pull request by MimimiBB-8**
- Tests are successfully passed

### CODE REVIEW

File: src/03-arrays-tasks.js [hunk 1]
```diff
@@ -83,8 +80,8 @@ function getArrayOfPositives(/* arr */) {
  *    [ 1, 2, 3, 4, 5 ] => []
  *    [ 'cat, 'dog', 'raccoon' ] => [ 'cat', 'dog', 'raccoon' ]
  */
-function getArrayOfStrings(/* arr */) {
-  throw new Error('Not implemented');
+function getArrayOfStrings(arr) {
+  return arr.filter((num) => typeof num === 'string');
```
Comment (xnj27) [inline src/03-arrays-tasks.js]: please, use el instead of num

File: src/03-arrays-tasks.js [hunk 2]
```diff
@@ -100,8 +97,8 @@ function getArrayOfStrings(/* arr */) {
  *    [ 1, 2, 3, 4, 5, 'false' ]         => [ 1, 2, 3, 4, 5, 'false' ]
  *    [ false, 0, NaN, '', undefined ]   => [ ]
  */
-function removeFalsyValues(/* arr */) {
-  throw new Error('Not implemented');
+function removeFalsyValues(arr) {
+  return arr.filter((num) => Boolean(num));
```
Comment (xnj27) [inline src/03-arrays-tasks.js]: num --> el
no need in Boolean()

File: src/03-arrays-tasks.js [hunk 3]
```diff
@@ -115,11 +112,12 @@ function removeFalsyValues(/* arr */) {
  *    => [ 'PERMANENT-INTERNSHIP', 'GLUTINOUS-SHRIEK', 'MULTIPLICATIVE-ELEVATION' ],
  *    [ 'a', 'b', 'c', 'd', 'e', 'f', 'g' ]  => [ 'A', 'B', 'C', 'D', 'E', 'F', 'G' ]
  */
-function getUpperCaseStrings(/* arr */) {
-  throw new Error('Not implemented');
+function getUpperCaseStrings(arr) {
+  let arr2 = [];
```
Comment (xnj27) [inline src/03-arrays-tasks.js]: you better name it "result" or something else, but not arr2, arr3 and etc.

**Review by xnj27 — APPROVED** [review]

File: src/03-arrays-tasks.js (hunk 1 above)
Comment (MimimiBB-8) [reply src/03-arrays-tasks.js]: argument renamed

File: src/03-arrays-tasks.js (hunk 2 above)
Comment (MimimiBB-8) [reply src/03-arrays-tasks.js]: argument renamed

File: src/03-arrays-tasks.js (hunk 3 above)
Comment (MimimiBB-8) [reply src/03-arrays-tasks.js]: delete 'result', return expression

**Pull request merged**